### DISCUSSION TITLE
Conditionally hide a field or select value

### DISCUSSION THREAD

**Discussion by FelixMalfait** [twentyhq/twenty#6990] [Ideas]
<img width="802" alt="Screenshot 2024-08-09 at 11 21 20" src="https://github.com/user-attachments/assets/f90bd7a5-06f2-46db-80d3-701876827fba">

<img width="654" alt="Screenshot 2024-08-09 at 11 22 40" src="https://github.com/user-attachments/assets/c5028dab-04fa-442f-be29-376c555a9b3e">

Related:
https://www.salesforcetutorial.com/field-dependency-in-salesforce/
https://developer.salesforce.com/files/accessibility/customize-a-salesforce-object/picklists-field-dependencies/index.html
https://trailhead.salesforce.com/content/learn/projects/customize-a-salesforce-object/picklists-field-dependencies

**Comment by FelixMalfait** [comment]
Should be done at the layout level. Converting to a discussion for when we build custom page layouts

**Comment by BOHEUS** [comment]
Related to https://github.com/twentyhq/twenty/discussions/16182
